scikeras/wrappers.py

```diff
@@ -1394,6 +1436,12 @@ def predict_proba(self, X):
             SciKeras will return an array of shape (n_samples, 2)
             (instead of `(n_sample, 1)` as in Keras).
         """
+        for k, v in kwargs.items():
+            warnings.warn(
+                "``**kwargs`` has been deprecated in SciKeras 0.2.1 and support will be removed be 1.0.0."
+                f" Instead, set predict_proba arguments at initialization (i.e., ``BaseWrapper({k}={v})``)"
```

Comment:
```suggestion
                f" Instead, set predict_proba arguments at initialization (i.e, ``BaseWrapper({k}={v})`` "
                f"or ``BaseWrapper(predict_proba__{k}={v})``"
```

Does that behavior work?

Reply:
No, for the purposes of parameter routing `predict_proba` uses `predict__` since they both call `predict__` in Keras. We could alias them if you think that might make things easier. It might also be good to have an error or at least a warning for unknown prefixes. All of this would have to be a separate PR.

Reply:
I've edited the suggested change to use the `predict__` prefix.